### PR TITLE
Third task

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
I have made a counter object which handles the counting.

### CODE REVIEW

File: src/multi/counter/Demo.java [hunk 1]
```diff
@@ -6,11 +6,13 @@
 public class Demo {
 
     public static void main(String[] args) {
-        MultiCounter controller = new MultiCounter(1, 5);
-        MultiCounter controlled = new MultiCounter(1, 10);
+        Counter count = new Counter("Thread2");
```
Comment (mgenov) [inline src/multi/counter/Demo.java]: Counter has single name but is used from 2 threads ?

File: src/multi/counter/Demo.java [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
+        MultiCounter controller = new MultiCounter("Thread1" ,1, 5, count);
```
Comment (mgenov) [inline src/multi/counter/Demo.java]: Why threads need to know for each other in this task ?

Comment (KrasimirRaikov) [inline src/multi/counter/Demo.java]: The description said that once the first thread reaches the end count, it should stop the other one, I don't know how to do this without the thread to know which is the other thread.

Comment (mgenov) [inline src/multi/counter/Demo.java]: ok

File: src/multi/counter/Demo.java (hunk 1 above)
Comment (KrasimirRaikov) [inline src/multi/counter/Demo.java]: I don't know if I can name objects with plural nouns

Comment (mgenov) [inline src/multi/counter/Demo.java]: I mean counter is named "Thread2".

Comment (KrasimirRaikov) [inline src/multi/counter/Demo.java]: Sorry, that wasn't the name of the counter, I just used the constructor to chose which thread should not start first, I will move it to a method.

Comment (KrasimirRaikov) [inline src/multi/counter/Demo.java]: Is it better now?

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**